modules/nested-elements/assets/js/editor/nested-repeater/controls/repeater.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * @extends ControlRepeaterItemView
+ */
+export default class Repeater extends elementor.modules.controls.Repeater {
+	className() {
+		// Repeater Panel CSS, depends on 'elementor-control-type-repeater` control.
+		// `elementor-control-type-nested-elements-repeater` to `elementor-control-type-repeater`
+		return super.className().replace( 'nested-elements-repeater', 'repeater' );
+	}
+
+	/**
+	 * @inheritDoc
+	 * Since the default is controlled via silent setSettings, override this method to avoid the default behavior.
+	 * Use command 'nested-elements/nested-repeater/select' instead.
+	 */
```

Comment:
```suggestion
	/**
	 * @inheritDoc
	 * Override to avoid the default behavior which applied via `editSettings.set( 'activeItemIndex', index )`.
	 * Since default behavior does not save history.
	 * Use command 'nested-elements/nested-repeater/select' instead.
	 */
```